apps/envited-x-data-space/modules/AddAssets/AddAssets.tsx

```diff
@@ -46,9 +46,13 @@ export const AddAssets = () => {
         times(idx => formData.append('assets', data.assets[idx]))(data.assets.length)
       }
 
-      await addAssetsForm(formData)
+      const results = await addAssetsForm(formData)
```

Comment:
While we're at it, the opposite is also true. addAssetsForm is actually doing a job but is declared as static. This should be a verb.